samples/idea-latest.yaml

```diff
@@ -0,0 +1,36 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: idea-latest
+spec:
+  started: false
+  template:
+    projects:
+      - name: web-nodejs-sample
+        git:
+          remotes:
+            origin: "https://github.com/vitaliy-guliy/web-nodejs-sample.git"
+            # origin: "https://github.com/che-samples/web-nodejs-sample.git"
+          checkoutFrom:
+            revision: "devfilev2"
+    components:
+      - name: dev
+        container:
+          image: quay.io/devfile/universal-developer-image:latest
+#        attributes:
+#          controller.devfile.io/merge-contribution: true
```

Comment:
No longer required due to https://github.com/devfile/devworkspace-operator/pull/1013

Reply:
Thanks!